verilog/CST/expression.cc

```diff
@@ -98,6 +98,19 @@ const verible::Symbol* GetConditionExpressionFalseCase(
   return GetSubtreeAsSymbol(condition_expr, NodeEnum::kConditionExpression, 4);
 }
 
+const verible::TokenInfo* GetUnaryPrefixOperator(
+    const verible::Symbol& symbol) {
+  const auto& children = verible::SymbolCastToNode(symbol).children();
+  const auto leaf_symbol = children.front().get();
```

Comment:
In general, we try to avoid using `auto` unless whatever comes back is a huge templated type which does not add any benefit for the reader. So use the type that is returned.

Reply:
Got it, thanks! Used auto because I saw it in use and thought that was the general style, looks much better now.